two_factor/admin.py

```diff
@@ -167,15 +149,15 @@ def wrapper(*args, **kwargs):
             return update_wrapper(wrapper, view)
 
         urlpatterns_2fa = [
-            url(r'^profile/$', wrap(self.two_factor_profile), name='profile'),
-            url(r'^profile/disable/$', wrap(self.two_factor_disable), name='disable'),
-            url(r'^setup/$', self.two_factor_setup, name='setup'),
-            url(r'^setup-complete/$', self.two_factor_setup_complete, name='setup_complete'),
-            url(r'^backup/tokens/$', wrap(self.two_factor_backup_tokens), name='backup_tokens'),
+            path('qrcode/', QRGeneratorView.as_view(), name='qr'),
+            path('^profile/', wrap(self.two_factor_profile), name='profile'),
+            path('^profile/disable/', wrap(self.two_factor_disable), name='disable'),
+            path('^setup/', self.two_factor_setup, name='setup'),
+            path('^setup-complete/', self.two_factor_setup_complete, name='setup_complete'),
+            path('^backup/tokens/', wrap(self.two_factor_backup_tokens), name='backup_tokens'),
```

Comment:
please, remove `^` from paths, they are excessive and produce warnings in the logs